.taskcluster.yml

```diff
@@ -206,6 +210,35 @@ tasks:
           owner: mcastelluccio@mozilla.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - $if: 'tasks_for == "github-pull-request" && "Train with task cluster:" in pr_description'
+        then:
+          dependencies:
+            - { $eval: as_slugid("docker_build") }
+          taskId: { $eval: as_slugid("train_on_taskcluster") }
+          created: { $fromNow: "" }
+          deadline: { $fromNow: "1 day" }
+          provisionerId: proj-bugbug
+          workerType: compute-smaller
+          payload:
+            maxRunTime: 10800
+            image: python:3.10
+            command:
+              - "/bin/sh"
+              - "-lcx"
+              - 'apt-get -qq update &&
+                apt-get -qq install -y zstd &&
+                git clone --quiet ${repository} &&
+                cd bugbug &&
+                git -c advice.detachedHead=false checkout ${head_rev} &&
+                pip install --disable-pip-version-check --no-cache-dir --progress-bar off -r requirements.txt &&
+                train_options=$(python -c "print(''''''${pr_description}''''''.rsplit('':'', 1)[-1].strip())") &&
```

Comment:
A regex to extract the model name would be more suitable. Also, it could be better to do it as a variable instead of `pr_description`.